pipewire/qubes-pw-module.c

```diff
@@ -187,7 +212,7 @@ struct qubes_stream {
     // Stream event listener.  Accessed on main thread only.
     struct spa_hook stream_listener;
     // Position control.  Pointer is immutable after being set.
-    // Pointee accessed on realtime thread only.
+    // Pointer accessed on realtime thread only.
```

Comment:
I used “pointee” to refer to what the pointer points to, as opposed to the pointer itself.

Reply:
Corrected, to be honest, I thought it was a typo without actually reading it as I did it at the very beginning of this work without understanding yet the whole context.